pkg/migrate/migrate.go

```diff
@@ -295,7 +295,7 @@ func cleanOldData(ctx context.Context, client *etcd.Client) {
 			if strings.HasPrefix(key, oldChangefeedPrefix) {
 				value = maskChangefeedInfo(kvPair.Value)
 			}
-			newKey := backupKeyPrefix + key
+			newKey := etcd.MigrateBackupKey(0, key)
```

Comment:
What does `0` mean here?

Reply:
0 is the version of the backup. For now, migration only supports version 0.